dingo/pipe/parser.py

```diff
@@ -99,6 +99,13 @@ def create_parser(top_level=True):
         help=("Number of calibration curves to use in marginalizing over calibration "
               "uncertainty"),
     )
+    calibration_parser.add(
+        "--calibration-correction-type",
+        type=str,
+        default="data",
+        help=("Number of calibration curves to use in marginalizing over calibration "
```

Comment:
Is this taken directly from the updated `bilby_pipe`? The help string seems wrong.

Do we need to update the `bilby_pipe` version?

Reply:
I just updated this to match the bilby_pipe helpstring, so it should be good to go. Also changed requirement to bilby_pipe == 1.4.2